### ISSUE TITLE
Use lodash-modularized packages instead of all of lodash

### ISSUE THREAD

**Issue by bendiy** [oozcitak/xmlbuilder-js#114]
An `npm install xmlbuilder` is 4.4MB because `xmlbuilder-js` includes all of lodash, which is 4.2MB. `xmlbuilder-js` is only using these pacakges:
https://www.npmjs.com/package/lodash.isobject
https://www.npmjs.com/package/lodash.isplainobject
https://www.npmjs.com/package/lodash.isfunction
https://www.npmjs.com/package/lodash.isempty
https://www.npmjs.com/package/lodash.assign
https://www.npmjs.com/package/lodash.camelcase
https://www.npmjs.com/package/lodash.kebabcase
https://www.npmjs.com/package/lodash.snakecase

```
npm install lodash.isobject lodash.isplainobject lodash.isfunction lodash.isempty lodash.assign lodash.camelcase lodash.kebabcase lodash.snakecase
```
Results in a `node_modules` of only 604KB.

**Comment by oozcitak** [comment]
We switched between single and modularized lodash in the past. See #43 and #53. Single lodash is huge, modularized lodash has a deep dependency tree.

Although I love lodash, I believe it is a bit heavy for xmlbuilder. And since our lodash dependency is relatively lightweight, I started implementing the above functions.

**Comment by bendiy** [comment]
Great! Libraries should have as few dependencies as possible. I love lodash too, but I'm trying to ship this to a browser and 4.2 MB is huge.

**Comment by oozcitak** [comment]
Fixed by ab0c6d3649828decfdd0c4b0e16021a3f300cc47
